### ISSUE TITLE
Vignette using multiple samples

### ISSUE THREAD

**Issue by lambdamoses** [pachterlab/voyager#25]
This is actually more like a research project, since it requires further consideration of what to make of spatial statistics across samples and their comparison in case vs. control. Also what it means to compute the spatial statistics jointly across samples or separately with each sample. Requires https://github.com/pachterlab/SpatialFeatureExperiment/issues/33

**Comment by abadgerw** [comment]
Just chiming in that this would be very helpful for data I have upcoming of spinal cord tissue.

We are considering mapping the tissue (and overlayed spots) of all patients to a standard space using VALIS to make it easier to perform a joint spatial analysis. Not sure if this would help make this easier? Would love to get your thoughts and see if our upcoming data may be useful in implementing methods to handle multiple samples.

**Comment by lambdamoses** [comment]
Yes, aligning the sections to a standard space will make it easier. Of course there are many open questions about ESDA for multiple samples that are unaddressed in the geospatial tradition. The vignette is a good place to discuss those questions.

**Comment by lambdamoses** [comment]
Another thing I want to explore: there's `sample_id` to perform joint analyses across samples while still keeping the spatial stuff distinguishable. The joint analyses can be non-spatial like clustering based on gene expression, or can be spatial like a hyper-global Moran's I. Compare the loadings from dimension reductions performed jointly and for different samples separately.

**Comment by dpschreiner** [comment]
many thanks for your efforts.  i see this is a fairly old issue, but chiming in anyway.

i think a very common, low-hanging-fruit use case (and probably what most biologists intuitively think spatial data is good for) would be the ability to compare results of colocalization of categorical variables (e.g. number/proportion of neighbors of a given cell type, or distance to some anatomical feature X) across conditions. in that case i'd assume the cell type annotation is done on all the data and the colocalization is obviously per sample. but we'd also want to report and handle effects arising due to different cell density.
